apps/drupal/patternlab.theme

```diff
@@ -39,6 +39,29 @@ function patternlab_preprocess_block(&$variables) {
     } else {
       $variables['logo_img_src'] = $logo_url;
     }
+  }
+
+  if ($id === 'system_powered_by_block') {
```

Comment:
Immediate thought here: should we explicitly prefer block-specific preprocess where they don't have common needs? That would help promote discoverability I think. E.g., `template_preprocess_block_system_branding_block()`.

Reply:
Fair, I'll swap this for these two blocks!